_posts/2021-01-12-december-rubygems-update.md

```diff
@@ -0,0 +1,48 @@
+---
+title: December 2020 RubyGems Updates
+layout: post
+author: Gift Egwuenu
+author_email: laurandidi21@gmail.com
+---
+
+Welcome to the RubyGems monthly update! As part of our efforts at [Ruby Together](http://rubytogether.org/), we publish a recap of the work that we’ve done the previous month. Read on to find out what updates were made to RubyGems and RubyGems.org in December.
+
+## RubyGems News
+
+In December, we finally released `bundler 2.2` and `rubygems 3.2` 🎉. We also received the corresponding feedback and regression reports, and addressed almost everything reported through 4 patch level releases of each library. In particular, we made it on time for Ruby's Christmas release and managed to include `rubygems 3.2.3` and `bundler 2.2.3` with the final release of `ruby 3.0`.
+
+To be mentioned apart, together with `rubygems 3.2.3` we released a fix to a longstanding gem install issue. From now on, gem install will respect the running ruby version, and install the latest version of the target gem that supports it. See [#4110](https://github.com/rubygems/rubygems/pulls/4110) for more information.
+
+Finally, we fixed a gem command that was crashing or asking for OTP twice when it prompted for an email and password. [#4162](https://github.com/rubygems/rubygems/pull/4162)
```

Comment:
I consider this as part of the " We also received the corresponding feedback and regression reports, and addressed almost everything reported through 4 patch level releases of each library." part, so I don't think it needs a special mention.

Reply:
Ah alright. Thought it was worth mentioning too. I'll fix that.

Reply:
I think the new feature is worth mentioning, specially because it requires coordination between server and client sides, which is something rubytogether is promoting greatly IMO. But I _think_ we mentioned it in previous updates? :thinking:

Reply:
Alright. So if I get this correctly. I should still add this -` Finally, we fixed a gem command that was crashing or asking for OTP twice when it prompted for an email and password. [#4162](https://github.com/rubygems/rubygems/pull/4162)` because I don't think we mentioned it in the previous update.

Reply:
Ok, then let me reword this mentioning the feature then, although maybe @sonalkr132 can word it better. I'd say the following, splitting the 1st paragraph into two:

> In December, we finally released bundler 2.2 and rubygems 3.2 :tada:. On the bundler side, this minor release provides some major enhancements in how bundler treats platforms, and also a few extra features. Check https://bundler.io/blog/2020/12/09/bundler-v2-2.html for details. On the rubygems side, the release provides a lot of bug fixes, a noticiable boot time speed up, better integration in ruby-core and alternative implementations, and adds support for a change in the server side that allows using scoped API keys.

> After the releases, we also received the corresponding feedback and regression reports,and addressed almost everything reported through 4 patch level releases of each library. In particular, we made it on time for Ruby's Christmas release and managed to include rubygems 3.2.3 and bundler 2.2.3 with the final release of ruby 3.0.

Reply:
Alright, this looks good to me.